### PR TITLE
rdm: blog june InvenioRDM release

### PR DESCRIPTION
**Pull request by fenekku**
June release blog post courtesy of @LisaOKeefe1 and @ppanero  . Take another quick look at it.

### CODE REVIEW

File: content/blog/2020-07-01-june-release/contents.lr [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+#### Creators, Contributors and Affiliations Identifiers
+
+The Deposit page now accepts [ORCID iDs](https://orcid.org/) for people (creators and contributors) and [ROR identifiers](https://ror.org/) for organizations (creators, contributors and affiliations). An icon with a link to the respective entity on the record landing page reflects this connection.
+
+Contributors are now optional on the deposit page as well.
+
+#### Python versions
+
+There are now Invenio images for Python 3.7 and Python 3.8! This means that you can use Python 3.6, 3.7 and 3.8 for development and for containerize.
```
Comment (jbenito3) [inline content/blog/2020-07-01-june-release/contents.lr:39]: for containerize?

Comment (ppanero) [reply content/blog/2020-07-01-june-release/contents.lr:39]: Is the concept we have (`--containerize`), meaning you are running the instance in containers.

Comment (jbenito3) [reply content/blog/2020-07-01-june-release/contents.lr:39]: Oh! Now I see. I guess while reading it sounded weird. Maybe if they were written like `development` and `containarize` or even including the `--` (`--development`...I would have guessed it :)

Comment (ppanero) [reply content/blog/2020-07-01-june-release/contents.lr:39]: Yeah that's true, and even if there is a new user that is not familiar to the `invenio-cli` that might sound weird... How could we rephrase it?

```suggestion
There are now Invenio images for Python 3.7 and Python 3.8! This means that you can use Python 3.6, 3.7 and 3.8 for local development and the full containerized option.
```
?

Comment (jbenito3) [reply content/blog/2020-07-01-june-release/contents.lr:39]: LGTM, thanks

**Comment by fenekku** [comment]
(someone who is authorized will have to merge this :) )

**Comment by jbenito3** [comment]
ups...clicked too fast :P

**Comment by ppanero** [comment]
@jbenito3 do you want me to ammend? or we leave as is?

**Pull request merged**